frontend/src/metabase/css/core/colors.css

```diff
@@ -3,7 +3,8 @@
   --brand-light-color: #CDE3F8;
   --brand-saturated-color: #2D86D4;
 
-  --base-grey: #f8f8f8;
+  --base-grey: #F8F9FA;
+  --grey-5percent: color(var(--base-grey) shade(5%));
```

Comment:
I don't really like that name but I don't know what's better.

Reply:
I know, it sucks.

Reply:
`grey-05`?

Reply:
`grey-05` seems like it might conform more to our existing setup.